### ISSUE TITLE
OCaml

### ISSUE THREAD

**Issue by goodmind** [yever/rusty-typescript#33]
It would be very cool if it would be easy to port this to OCaml, at least lexer and parser

**Comment by lulu-berlin** [comment]
Port the Rust code? What's the use case? I think we should first finish porting it from TypeScript to Rust.

**Comment by goodmind** [comment]
@yever yeah, or bindings to OCaml with C interface. It is just impossible to use TypeScript parser in other languages

**Comment by goodmind** [comment]
Use case is to merge TypeScript into Flow 😳
Aka write AST-to-AST converter directly in Flow typechecker for .ts file, this is just mad idea I had.
It would be lossy conversion of course

**Comment by lulu-berlin** [comment]
Well, part of the idea here, as I tried to outline in #24 is to create a Rust library for compiling TypeScript separately from the WebAssembly target. This Rust library could then have bindings to whichever language you want. Additionally, the WebAssembly target can be integrated anywhere you have a browser or nodejs.
